auto-export.config.json

```diff
@@ -1,9 +1,9 @@
 {
     "entries": [
         {
-            "inputPath": "packages/core/src/app/paymentIntegration/**/index.ts",
-            "outputPath": "packages/core/src/app/generated/paymentIntegrations/index.ts",
-            "memberPattern": ".+PaymentMethod$"
+            "inputPath"     : "packages/**/src/index.ts",
```

Comment:
is there a reason for the spaces added?

Reply:
After addition, it would be eaiser to read and compare those paths.

Reply:
I prefer not aligning JSON keys because we don't want to re-align everything when we add a new key that is longer than the current longest key.

Reply:
oH, I see.